shared/utility/io/uart.cpp

```diff
@@ -10,10 +10,7 @@
 #include <unistd.h>
 
 namespace utility::io {
-
-    uart::uart() : device(""), fd(-1) {}
```

Comment:
Why did we get rid of the default constructor?

Reply:
I set `uart() = default;` in the .hpp